lang/es.json

```diff
@@ -83,7 +81,7 @@
   "AddFundsModal.tipAndFinish": "Dar propina y terminar",
   "AddFundsModal.YouAdded": "Añadiste:",
   "administrators": "Administradores",
-  "AdminPanel.button": "Administrador",
+  "AdminPanel.button": "Admin",
```

Comment:
The next Crowdin update would solve it, but we're resetting the translation there. Quick fix:

```bash
git checkout origin/main lang/*
npm run build:langs
```